src/SSW.Rewards.Application/Quizzes/Commands/AddNewQuiz/AddNewQuizCommandValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace SSW.Rewards.Application.Quizzes.Commands.AddNewQuiz;
+public class AddNewQuizCommandValidator : AbstractValidator<AddNewQuizCommand>
+{
+    private readonly IApplicationDbContext _context;
+
+    public AddNewQuizCommandValidator(IApplicationDbContext context)
+    {
+        _context = context;
+
+        RuleFor(c => c)
+            .MustAsync(BeUniqueQuiz);
+    }
```

Comment:
Add checks for...
* No questions
* Empty question text
* No answers for questions
* Empty answer text
* More than 1 correct answer per question
* No correct answers per question

Reply:
Done